components/list/list-item-generic-layout.js

```diff
@@ -225,20 +237,33 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 		}
 	}
 
+	_focusFirstCell() {
```

Comment:
In a previous attempt, I changed the `data-cell-num` from 1 through 8 to -5 through 2, which meant I had to make other changes to handle that. It shouldn't be necessary anymore, but I don't hate starting at 0 to indicate where focus starts so I left it in for now.

Reply:
I had previously moved things around so `content-actions` was not the first cell in the DOM order and I was retargetting focus to first cell (which was `6`) manually on `focusin`, and at the point neither the DOM order or the cell numbers really made any sense, so I wanted to be able to say `_focusCellItem(0)` and have it go to our starting position.

However, I agree with reverting to the original.